addons/sale_timesheet/controllers/main.py

```diff
@@ -18,13 +18,41 @@ def plan(self, domain):
         }
 
     def _prepare_plan_values(self, domain):
+
```

Comment:
Side note: could we remove the request.env['ir.model.data'].get_object in the legacy code just above this change :p ?

Reply:
ok